### DISCUSSION TITLE
installing VS 17.4 Preview 2.1 breaks 17.3?

### DISCUSSION THREAD

**Discussion by wallym** [dotnet/maui#10240] [Q&A]
I installed the VS 17.4 Preview 2.1 update yesterday late afternoon.  It seems to have wiped out my Maui setup in VS 17.3.  Did I miss this in some readme file or list of known issues?  I have a Maui app that I am working on based on .net 6 maui using VS 17.3.x.  Everything worked until I updated yesterday afternoon.  I am now running on 17.4 P2.1.  This isn't a big deal at this moment, just wondering why this happened.  The error message I am getting in 17.3 is "Platform version is not present for one or more target frameworks, even though they have specified a platform: net6.0-ios, .net6.0-maccatalyst."  Any ideas are appreciated.

Wally

**Comment by danielcooper-hk** [comment]
I had exactly the same issue with this as well and I couldn't fix it.  Ended up with a clean install on my dev computer and leaving out the preview version.  I lost a whole day trying to fix this and it came at a really inconvenient time.

**Reply by wallym** [reply]
I uninstalled my preview and current 2022.  I reinstalled VS 2022.  Same problem still.

**Comment by wallym** [comment]
global.json seems to solve the problem.  https://learn.microsoft.com/en-us/dotnet/core/tools/global-json
